lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerCsvTest.kt

```diff
@@ -30,547 +35,412 @@ class LapisControllerCsvTest(@Autowired val mockMvc: MockMvc) {
     @MockkBean
     lateinit var siloQueryModelMock: SiloQueryModel
 
-    val listOfMetadata = listOf(
-        DetailsData(mapOf("country" to TextNode("Switzerland"), "age" to IntNode(42))),
-        DetailsData(mapOf("country" to TextNode("Switzerland"), "age" to IntNode(43))),
-    )
-
-    val metadataCsv = """
-        country,age
-        Switzerland,42
-        Switzerland,43
-    """.trimIndent()
-
-    val metadataTsv = """
-        country	age
-        Switzerland	42
-        Switzerland	43
-    """.trimIndent()
-
-    val aggregationData = listOf(
-        AggregationData(
-            0,
-            mapOf("country" to TextNode("Switzerland"), "age" to IntNode(42)),
-        ),
-    )
-
-    val aggregationDataCsv = """
-        country,age,count
-        Switzerland,42,0
-    """.trimIndent()
-
-    val aggregationDataTsv = """
-        country	age	count
-        Switzerland	42	0
-    """.trimIndent()
-
-    val nucleotideMutationData = listOf(
-        NucleotideMutationResponse(
-            "sequenceName:1234",
-            2345,
-            0.987,
-        ),
-    )
-
-    val aminoAcidMutationData = listOf(
-        AminoAcidMutationResponse(
-            "sequenceName:1234",
-            2345,
-            0.987,
-        ),
-    )
-
-    val mutationDataCsv = """
-        mutation,count,proportion
-        sequenceName:1234,2345,0.987
-    """.trimIndent()
+    @ParameterizedTest(name = "GET empty {0} returns empty data")
+    @MethodSource("getEndpoints")
+    fun `GET empty returns empty json`(
```

Comment:
Strictly speaking, the GET is not empty, only the response is empty.
```suggestion
    fun `GET returns empty json`(
```